### DISCUSSION TITLE
Any plans to update the 'Physical Entities'?

### DISCUSSION THREAD

**Discussion by sdotlock** [axelboc/anki-ultimate-geography#662] [General]
I explain what this deck is to people and then say that I've finished it, then get shamed when they say, "Where is the Andaman sea?" and I cant tell them. According to the 'Physical Entities' list last updated three years ago it should be included, but still isn't. I got bullied out of a party because I had never heard of the Ross Sea. 

What are the plans to fix up the seas/physical entities etc? Or have I missed something?

**Comment by aplaice** [comment]
Apologies that my procrastination lead to your being bullied! :D 

The main reasons are:

1. I haven't had time to finish off the maps (we're generating them ourselves, because unfortunately Wikipedia/Wikimedia, while having amazing maps for countries, doesn't really have matching ones for sea bodies).
2. This is complicated by the fact that the Natural Earth data ("shapefiles") that we've been using to generate the maps don't line up with the IHO 1953 or the IHO 2002 definitions that we're relying on. For the maps that have already been generated and are in the deck, the discrepancies are sufficiently tiny that we've let them go (see https://github.com/anki-geo/ultimate-geography/discussions/412). For many (most? I haven't looked at this in a while) of the ones that haven't, they are considerably greater, so it feels wrong to generate the maps from these shapefiles. I have long had ideas of parsing the IHO 2002 definition to generate shapefiles from scratch (probably using LLMs or possibly very fancy regexes), but this is non-trivial because they're plain text descriptions.
